src/v/kafka/requests/fetch_request.cc

```diff
@@ -302,12 +310,10 @@ static ss::future<read_result> read_from_partition(
       std::nullopt);
 
     reader_config.strict_max_bytes = config.strict_max_bytes;
-    auto hw = pw.high_watermark();
-    auto lso = pw.last_stable_offset();
-
-    return pw.make_reader(reader_config, deadline)
+    return pw.make_reader(reader_config)
       .then([hw, lso, foreign_read, deadline](model::record_batch_reader rdr) {
-          // if we are on remote core, we MUST use foreign record batch reader.
```

Comment:
seems like a bad formatting ? you can push `reader` one line above

Reply:
done